helm-app/values.yaml

```diff
@@ -0,0 +1,160 @@
+apiGateway:
+  name: api-gateway
+  namespace: my-app-namespace-3
+  replicas: 1
+  containerName: api-gateway
+  image:
+    repository: imzainazm/api-gateway
+    tag: latest
+  containerPort: 7000
+  service:
+    name: api-gateway-service
+    type: NodePort
+    port: 7000
+    targetPort: 7000
+    nodePort: 32000
+
+chatApp:
+  name: chat-app
+  namespace: my-app-namespace-3
+  replicas: 1
+  containerName: chat-app
+  image:
+    repository: imzainazm/chat-app
+    tag: latest
+  containerPort: 7001
+  service:
+    name: chat-app-service
+    type: NodePort
+    port: 7001
+    targetPort: 7001
+    nodePort: 32001
+  configMap:
+    name: app-config
+    key: API_GATEWAY_URI
+    apiGatewayUri: "http://172.29.25.105:32000"
```

Comment:
Why is this hardcoded?

Reply:
Hardcoded the Node Ip and Node port of minikube for intial setup.

Reply:
Could you explain why do you need to do this?

Reply:
yes, we don't need to do this. It is suitable with service-name : port (k8s internal server DNS feature)